portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,24 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> infoAboutMe = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Awad!</h1>");
+    //populate list
+    infoAboutMe.add("My name is Awad.");
+    infoAboutMe.add("I'm 19 years old.");
+    infoAboutMe.add("I'm a STEP Intern for Google.");
+    String jsonInfo = convertToJsonUsingGson();
+    //sends Json as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(jsonInfo);
+
   }
+
+  private String convertToJsonUsingGson(){
+    Gson gson = new Gson();
```

Comment:
Do you need to be making a new Gson object every time, or can this be initialized once and reused?

Reply:
Didn't think of that, thanks for pointing that out!